release-notes/opensearch-release-notes-1.0.0-rc1.md

```diff
@@ -0,0 +1,334 @@
+# OpenSearch and Dashboards 1.0.0-rc1 Release Notes
+
+## Release Highlights
```

Comment:
Suggested text:

This Release Candidate is a version of the project that is feature complete and passing automated testing with the intent to validate expected functionality before moving to a General Availability (GA) launch. There are multiple enhancements and fixes that are part of this release including span filtering support in Trace Analytics, tenant support in Notebooks, K-NN field level algorithm selection, support for index management transforms, and support for scheduling and tenants in reporting.

Reply:
Added. Thanks